test/conformance/generatename_test.go

```diff
@@ -0,0 +1,187 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	pkgTest "github.com/knative/pkg/test"
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	rtesting "github.com/knative/serving/pkg/reconciler/v1alpha1/testing"
+	"github.com/knative/serving/test"
+	"net/http"
```

Comment:
Separate standard library packages from other packages. For example,

```
import (
    "net/http"
    "strings"
    ...

    "github.com/knative/servingpkg/apis/serving/v1alpha1"
    "github.com/knative/serving/test
    ...
```

Reply:
Done, thanks.